rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,338 @@
+
+
+@pytest.mark.parametrize('event_type,event_subtype,is_taxable,counterparty,gas_taxable', [
+    (HistoryEventType.SPEND, HistoryEventSubType.NONE, True, None, False),
+    (HistoryEventType.SPEND, HistoryEventSubType.FEE, True, None, False),
+    (HistoryEventType.SPEND, HistoryEventSubType.FEE, True, CPT_GAS, True),
+    (HistoryEventType.SPEND, HistoryEventSubType.FEE, False, CPT_GAS, False),
+    (HistoryEventType.DEPOSIT, HistoryEventSubType.NONE, False, None, False),
+    (HistoryEventType.RENEW, HistoryEventSubType.NONE, True, None, False),
+])
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+def test_accounting_spents_settings(
+        accounting_pot: 'AccountingPot',
+        event_type: 'HistoryEventType',
+        event_subtype: 'HistoryEventSubType',
+        is_taxable: bool,
+        counterparty: Optional[str],
+):
+    """Test that the default accounting settings for spends are correct"""
+    # Gain some ETH so that the spending event has something to spend
+    _gain_one_ether(transactions=accounting_pot.transactions)
+
+    spend_event = HistoryBaseEntry(
+        event_identifier=EXAMPLE_EVM_HASH,
+        sequence_index=0,
+        timestamp=TIMESTAMP_2_MS,
+        location=Location.BLOCKCHAIN,
+        location_label=EXAMPLE_ADDRESS,
+        asset=A_ETH,
+        balance=Balance(amount=ONE),
+        notes='Send 1 ETH to 0xABC',
+        event_type=event_type,
+        event_subtype=event_subtype,
+        counterparty=counterparty,
+    )
+    consumed_num = accounting_pot.transactions.process(event=spend_event, events_iterator=iter([]))
+    assert consumed_num == 1
+
+    acquisition_event = AssetAcquisitionEvent(
+        amount=ONE,
+        timestamp=TIMESTAMP_1_SECS,
+        rate=Price(FVal(1862.06)),
+        index=0,
+    )
+    acquisition_event.remaining_amount = ZERO
+    cost_basis = None
+    if is_taxable:
+        cost_basis = CostBasisInfo(
+            taxable_amount=ONE,
+            taxable_bought_cost=FVal(1862.06),
+            taxfree_bought_cost=ZERO,
+            matched_acquisitions=[
+                MatchedAcquisition(
+                    amount=ONE,
+                    event=acquisition_event,
+                    taxable=True,
+                ),
+            ],
+            is_complete=True,
+        )
+    expected_event = ProcessedAccountingEvent(
+        type=AccountingEventType.TRANSACTION_EVENT,
+        notes='Send 1 ETH to 0xABC',
+        location=Location.BLOCKCHAIN,
+        timestamp=TIMESTAMP_2_SECS,
+        asset=A_ETH,
+        free_amount=ZERO,
+        taxable_amount=ONE,
+        price=Price(FVal(3258.55)),
+        pnl=PNL(),
+        cost_basis=cost_basis,
+        index=1,
+        extra_data={'tx_hash': EXAMPLE_TX_HASH_HEX},
+    )
+    expected_event.calculate_pnl(
+        count_entire_amount_spend=is_taxable,
+        count_cost_basis_pnl=is_taxable,
+    )
+    assert accounting_pot.processed_events[-1] == expected_event
```

Comment:
again same thing. You don't seem to check the PnL at all. The entire point of accounting is to have correct PnL. This is what you should check